lib/lit/loader.rb

```diff
@@ -10,7 +10,6 @@ class Loader
     attr_accessor :logger
     def initialize
       self.logger ||= Logger.new($stdout)
-      self.logger.info 'initializing Lit'
```

Comment:
actually I'd prefer leaving this, at least for non-test environments.

Reply:
reverted